docs/mintlify-src/datastore/computed-columns.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: 'Computed Columns'
```

Comment:
Is it possible to do a `git mv` on the original mintlify folder to relocate the existing files to `mintlify-src`? Making a fresh copy of them loses the original commit history.

Reply:
done